sdk/eventhubs/azure_messaging_eventhubs/README.md

```diff
@@ -1,7 +1,302 @@
-# azure_messaging_servicebus
+<!-- cspell:words eventhubs  -->
 
-Azure Event Hubs crate for the Microsoft Azure SDK for Rust.
+# Azure Event Hubs Client Package for Rust
 
-This crate is part of a collection of crates: for more information please refer to [https://github.com/azure/azure-sdk-for-rust](https://github.com/azure/azure-sdk-for-rust).
+Azure Event Hubs crate for the Microsoft Azure SDK for Rust.
 
 License: MIT
+
+[Azure Event Hubs](https://azure.microsoft.com/services/event-hubs/) is a big data streaming platform and event ingestion service from Microsoft. For more information about Event Hubs see: [link](https://docs.microsoft.com/azure/event-hubs/event-hubs-about).
+
+Use the client library `` in your application to:
+
+-   Send events to an event hub.
+-   Consume events from an event hub.
+
+Key links:
+
+-   [Source code][source]
+-   [API Reference Documentation][rustdoc]
+-   [Product documentation](https://azure.microsoft.com/services/event-hubs/)
+-   [Samples][rustdoc_examples]
+
+## Getting started
+
+### Install the package
+
+Add the Azure Event Hubs client package for rust to your `cargo.toml` file:
+
+```toml
+[dependencies]
+azure_messaging_eventhubs = { version="*" }
```

Comment:
This is not good advice. It should be explicit, or at least a major and/or minor range. Alternatively, just tell them to run `cargo add azure_messaging_eventhubs`...which will work someday once we publish, which what you have won't work now anyway.

Reply:
I'll change it to cargo add.